js/coinsbit.js

```diff
@@ -0,0 +1,1103 @@
+            'amount': amount,
+            'cost': cost,
+            'fee': fee,
+        }, market);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+        };
+        let method = undefined;
+        if (type === 'limit') {
+            request['side'] = side;
+            request['price'] = this.priceToPrecision (symbol, price);
+            request['amount'] = this.amountToPrecision (symbol, amount);
+            method = 'privatePostOrderNew';
+        } else if (type === 'market') {
+            request['direction'] = side;
+            method = 'privatePostOrderNewMarket';
+            // for market buy it requires the amount of quote currency to spend
+            if (side === 'buy') {
+                let cost = undefined;
+                if (this.options['createMarketBuyOrderRequiresPrice']) {
+                    if (price === undefined || amount === undefined) {
+                        throw new InvalidOrder (this.id + " createOrder() requires the price argument for market buy orders to calculate total order cost (amount to spend), where cost = amount * price. Supply a price argument to createOrder() call if you want the cost to be calculated for you from price and amount, or, alternatively, add .options['createMarketBuyOrderRequiresPrice'] = false and supply the total cost value in the 'amount' argument or in the 'amount' extra parameter (the exchange-specific behaviour)");
+                    }
+                    cost = amount * price;
+                } else {
+                    cost = amount;
+                }
+                request['amount'] = this.costToPrecision (symbol, cost);
+            } else if (side === 'sell') {
+                request['amount'] = this.amountToPrecision (symbol, amount);
+            }
+        }
+        const response = await this[method] (this.extend (request, params));
+        // {
+        //     success: true,
+        //     message: '',
+        //     result: {
+        //       orderId: '8629354782',
+        //       market: 'ETH_USDT',
+        //       price: '4000',
+        //       side: 'buy',
+        //       type: 'limit',
+        //       timestamp: '1640212275.928',
+        //       dealMoney: '7.98856357',
+        //       dealStock: '0.002',
+        //       amount: '0.002',
+        //       takerFee: '0.002',
+        //       makerFee: '0.002',
+        //       left: '0',
+        //       dealFee: '0'
+        //     },
+        //     code: '200'
+        //   }
+        const data = this.safeValue (response, 'result', {});
+        return this.parseOrder (data, market);
+    }
+
+    async cancelOrder (id, symbol = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new InvalidOrder (this.id + ' cancelOrder needs symbol parameter');
```

Comment:
Let's change this error message to `this.id + ' cancelOrder requires a symbol argument');`